### DISCUSSION TITLE
Is there a way to get notifications when someone mentioned me in release notes

### DISCUSSION THREAD

**Discussion by kyu08** [community/community#136165] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

## My issue
I want to know when my PR is released when I contribute to OSS.
I know that I can get notified if I watch the repository, but regarding OSS, I don't want to do that because I want to get notified the release only I related.

## Solution I thought of
Usually, contributors are mentioned in the release notes, so I thought if you can receive notifications when you are mentioned in the release notes, this issue will be resolved.
If there is a way to do this, I would be greatful if you let me know.

**Comment by HAMZA-420** [comment]


**Reply by kyu08** [reply]
I don't feel like your answer follows [the code of conduct of community/community](https://github.com/community/community/blob/fc052e3e356d7fe1df0f92529e9c36be933541f4/CODE_OF_CONDUCT.md).

> - Take personal responsibility for everything you post.
> - Use your expertise as a developer to verify that the answer works and makes sense.

How about check it out?

**Comment by rikeshdahal** [comment]
i got same
